printer_mods/rsheng118/BTT_octopus_DIN_vertical_mount/readme.md

```diff
@@ -0,0 +1,14 @@
+# BTT octopus vertical mount
+
+## Overview
+This is a mount for new BTT octopus mount for rail. it mounts the board vertically to take full advantage of the fan to cool those stepper motor
```

Comment:
Capitalize `Octopus`.

Replace `for rail` with `for DIN rails`

Capitalize `It` at the start of the sentence.

Replace `stepper motor` with `stepper motor drivers`.

Reply:
Still need to capitalize Octopus in this sentence, and replace `rail` with `rails`.